src/main/java/edu/harvard/iq/dataverse/harvest/server/web/servlet/OAIServlet.java

```diff
@@ -192,9 +194,13 @@ private RepositoryConfiguration createRepositoryConfiguration() {
         // (Note: if the setting does not exist, we are going to assume that they
         // have a reason not to want to configure their email address, if it is
         // a developer's instance, for example; or a reason not to want to 
-        // advertise it to the world.) 
-        InternetAddress systemEmailAddress = MailUtil.parseSystemAddress(settingsService.getValueForKey(SettingsServiceBean.Key.SystemEmail));
-        String systemEmailLabel = systemEmailAddress != null ? systemEmailAddress.getAddress() : "donotreply@localhost";
+        // advertise it to the world.)
+        String systemEmailLabel = "donotreply@localhost";
```

Comment:
I'm not sure if this is a wise default value.

Reply:
Well, it's existing. A pre-existing condition from before this pull request. 😄

Reply:
I agree - it's a bad choice. But we must be backward compatible.